src/solver/conv_cellfft.cpp

```diff
@@ -47,517 +46,350 @@
 
 static const char* g_knames[]=
 {
-    "sfft4x4_r2c_perm_s3x3",
-    "sfft4x4_r2c_perm_s5x5",
-    "sfft4x4_r2c_qerm_s3x3",
-    "sfft4x4_r2c_qerm_s5x5",
-    "sfft5x5_r2c_perm_s3x3",
-    "sfft5x5_r2c_perm_s5x5",
-    "sfft5x5_r2c_qerm_s3x3",
-    "sfft5x5_r2c_qerm_s5x5",
+    "sfft4x4_r2c_perm_s3x3", "sfft4x4_r2c_perm_s5x5",
```

Comment:
I think you want to insert `//clang-format off` here and revert to linear list.

Reply:
> I think you want to insert `//clang-format off` here and revert to linear list.

Have changed.